cmd/gpu-operator/main.go

```diff
@@ -104,13 +105,23 @@ func main() {
 		Port: 9443,
 	})
 
+	operatorNamespace := os.Getenv("OPERATOR_NAMESPACE")
+	openshiftNamespace := "openshift"
+	cacheOptions := cache.Options{
+		DefaultNamespaces: map[string]cache.Config{
+			operatorNamespace:  {},
+			openshiftNamespace: {},
```

Comment:
nit: can we add a comment denoting that this is needed for getting / listing `ImageStream` resources in the `openshift` namespace?

Reply:
For sure!